fitting/fitScanData.py

```diff
@@ -84,25 +170,28 @@ def feed(self, event):
                 if(event.vcal > 254):
                     self.scanCount[event.vfatN][event.vfatCH] += event.Nhits
             else:
-                charge = self.calDAC2Q_m[event.vfatN]*(event.vcal)+self.calDAC2Q_b[event.vfatN]
+                charge = self.calDAC2Q_m[event.vfatN]*(256-event.vcal)+self.calDAC2Q_b[event.vfatN]
                 if((256-event.vcal) > 254):
                     self.scanCount[event.vfatN][event.vfatCH] += event.Nhits
         else:
             if(event.vcal > 250):
                 self.scanCount[event.vfatN][event.vfatCH] += event.Nhits
-                pass
-            pass
-
-        from gempython.gemplotting.utils.anautilities import first_index_gt
-        from math import sqrt
-        chargeBin = first_index_gt(self.scanHistosChargeBins[event.vfatN][event.vfatCH], charge)-1
-        self.scanHistos[event.vfatN][event.vfatCH].SetBinContent(chargeBin,event.Nhits)
-        self.scanHistos[event.vfatN][event.vfatCH].SetBinError(chargeBin,sqrt(event.Nhits))
+        self.scanHistos[event.vfatN][event.vfatCH].Fill(charge,event.Nhits)
```

Comment:
This appears to be incorrect; I think you merged/rebased from a bad starting point.